src/ReactSVG.tsx

```diff
@@ -70,12 +73,16 @@ export class ReactSVG extends React.Component<Props, State> {
       const wrapper = this.props.wrapper!
       /* eslint-enable @typescript-eslint/no-non-null-assertion */
 
-      const nonReactElement = document.createElement(wrapper)
+      const nonReactElement =
+        wrapper === 'svg'
+          ? document.createElementNS(svgNamespace, wrapper)
```

Comment:
Since this is always going to be an inner SVG element, no need for the `xmlns` & `xmlns:xlink` attrs (see [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg)).